incubator/patroni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: patroni
 description: "Highly available elephant herd: HA PostgreSQL cluster."
-version: 0.2.0
+version: 0.2.1
```

Comment:
Since the volume template section changed, upgrading this chart will break people's installations (I guess). Should we rather bump the minor version?